app/Widgets/Twitter/Views/twitter_widget.blade.php

```diff
@@ -0,0 +1,33 @@
+<div class="card">
+    <div class="card-body">
+        <h3 class="card-title">{{ ucwords(trans('twitter::widget.title')) }}</h3>
+
+        <table class="table table-striped table-sm">
+            <thead>
+                <tr>
+                    <th>Name</th>
+                    <th>Tweet Count</th>
+                </tr>
+            </thead>
+            <tbody>
+                @foreach($tweets as $val)
+                    <tr>
+                        <td><a href="http://twitter.com/search?q={{$val->tweet_name}}" target="_blank">{{$val->tweet_name}}</a></td>
+                        @if($val->tweet_count > 0)
+                            <td>{{$val->tweet_count}}</td>
```

Comment:
Can you add a `number_format()` for the tweet count so it adds appropriate commas? You may need to do a check to be sure it is not a **-** before adding the formatting it.